Makefile

```diff
@@ -50,22 +50,24 @@ KUBECFG_CMD = $(DOCKER) run \
 	$(KSONNET_BUILD_IMAGE) \
 	kubecfg show -o yaml -V RBAC_ENABLED=$(RBAC_ENABLED) -J $(WORKDIR) -o yaml $< > $@
 
+DOCKER_RUN ?= $(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c
+
 .PHONY: all container push clean cbuild test local generate
 
 all: container
 
 test:
-	$(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c '$(TEST)'
+	 $(DOCKER_RUN) '$(TEST)'
```

Comment:
want to have the ```test: cbuild``` 

and you could get rid of the other stage above.

Reply:
I don't quite understand this. They are running different commands?

Reply:
They are, but you probably shouldn't run the tests if the builds fail. 

So making it a dependent target makes sense to me.

Reply:
legit!